docs/requirements.yaml

```diff
@@ -87,3 +87,6 @@ dependencies:
   - pandoc
   - sphinx
   - sphinx_rtd_theme
+
+  - pip:
+      - sphinx-collections
```

Comment:
must resist urge to make conda-forge package...

Reply:
You should do it!  It's a very useful package.